packages/react-ui-ag/src/Drawer/Drawer.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { Motion, spring } from 'react-motion'
+import themed from 'react-themed'
+import PropTypes from 'prop-types'
+import { ToggleButton } from '@rentpath/react-ui-core'
+import classnames from 'classnames'
+import autobind from 'autobind-decorator'
+
+@themed(/^Drawer/, {
+  pure: true,
+})
+
+export default class Drawer extends React.Component {
```

Comment:
Is it possible to follow the same syntax as the other components? i.e. `extends Component {`?